### ISSUE TITLE
FieldTypeAsString not accepted during deserialization

### ISSUE THREAD

**Issue by Askelkana** [pipedrive/client-php#97]
Stack trace:

```
PHP Fatal error:  Uncaught InvalidArgumentException: Invalid value for enum '\Pipedrive\Model\FieldTypeAsString', must be one of: 'address', 'date', 'daterange', 'double', 'enum', 'monetary', 'org', '
people', 'phone', 'set', 'text', 'time', 'timerange', 'user', 'varchar', 'varchar_auto', 'visible_to' in ...\vendor\pipedrive\pipedrive\lib\ObjectSerializer.php:375
Stack trace:
#0 ...\vendor\pipedrive\pipedrive\lib\ObjectSerializer.php(400): Pipedrive\ObjectSerializer::deserialize('int', '\\Pipedrive\\Mode...', NULL)
#1 ...\vendor\pipedrive\pipedrive\lib\ObjectSerializer.php(287): Pipedrive\ObjectSerializer::deserialize(Object(stdClass), '\\Pipedrive\\Mode...', NULL)
#2 ...\vendor\pipedrive\pipedrive\lib\ObjectSerializer.php(400): Pipedrive\ObjectSerializer::deserialize(Array, '\\Pipedrive\\Mode...', NULL)
#3 ...\vendor\pipedrive\pipedrive\lib\Api\DealFieldsApi.php(1373): Pipedrive\ObjectSerializer::deserialize(Object(stdClass), '\\Pipedrive\\Mode...', Array)
#4 ...\vendor\pipedrive\pipedrive\lib\Api\DealFieldsApi.php(1314): Pipedrive\Api\DealFieldsApi->getDealFieldsWithHttpInfo(0, NULL)
```

**Comment by arturludan** [comment]
I have the same problem for field types like "int" and "picture".

**Comment by jonghyeok** [comment]
I also same problem when call `getDealFields` function.

- Pipedrive version: 6.2.0

Use same example code
```
$apiInstance = new Pipedrive\Api\DealFieldsApi(
    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
    // This is optional, `GuzzleHttp\Client` will be used as default.
    new GuzzleHttp\Client(),
    $config
);
$start = 0; // int | Pagination start
$limit = 100; // int | Items shown per page

try {
    $result = $apiInstance->getDealFields($start,$limit);
    echo '<pre>';
    print_r($result);
    echo '</pre>';
} catch (Exception $e) {
    echo 'Exception when calling DealFieldsApi->getDealFields: ', $e->getMessage(), PHP_EOL;
}
```

Then code return this error.
`Exception when calling DealFieldsApi->getDealFields: Invalid value for enum '\Pipedrive\Model\FieldTypeAsString', must be one of: 'address', 'date', 'daterange', 'double', 'enum', 'monetary', 'org', 'people', 'phone', 'set', 'text', 'time', 'timerange', 'user', 'varchar', 'varchar_auto', 'visible_to'`

**Comment by s-pogoda** [comment]
Hello @Askelkana @arturludan @jonghyeok,

I'm writing to let you know, that the issue with getting Pipedrive entity fields now fixed in the `7.1.0` version

Sorry for the delayed replay
